### ISSUE TITLE
Spawning broken?

### ISSUE THREAD

**Issue by smessmer** [rustyscreeps/screeps-game-api#275]
Expected behavior:
------
This code is supposed to output the currently spawning creep to the console:
```
for spawn in screeps::game::spawns::values() {
    if let Some(spawning) = spawn.spawning() {
        info!("Currently spawning {}", spawning.name());
    }
}
````

Actual behavior:
-----
```
[11:49:19 PM]caught exception: ReferenceError: Spawning is not defined
[11:49:19 PM]stack trace: ReferenceError: Spawning is not defined
    at __cargo_web_snippet_779245b618151baf119980cea9432ba3a28d6bc8 (eval at exports.evalCode (blob:https://screeps.com/cdbe6797-ff3f-4ad1-a7dd-efcc70585ed9:2:11476), :995:95)
    at _ZN7screeps10game_loop_17h0751b4cb239d4d2bE (wasm-function[67]:0x3198)
    at _ZN188_$LT$stdweb..webcore..newtype..Newtype$LT$$LP$stdweb..webcore..serialization..FunctionTag$C$$u20$$LP$$RP$$RP$$C$F$GT$$u20$as$u20$stdweb..webcore..serialization..FuncallAdapter$LT$F$GT$$GT$15funcall_adapter17ha50fcf942c5cad81E (wasm-function[64]:0x2731)
    at Object.Module.STDWEB_PRIVATE.dyncall (eval at exports.evalCode (blob:https://screeps.com/cdbe6797-ff3f-4ad1-a7dd-efcc70585ed9:2:11476), :458:40)
    at output (eval at exports.evalCode (blob:https://screeps.com/cdbe6797-ff3f-4ad1-a7dd-efcc70585ed9:2:11476), :157:39)
    at Object.module.exports.loop (eval at exports.evalCode (blob:https://screeps.com/cdbe6797-ff3f-4ad1-a7dd-efcc70585ed9:2:11476), :998:108)
    at Object.wasm_initialize (eval at exports.evalCode (blob:https://screeps.com/cdbe6797-ff3f-4ad1-a7dd-efcc70585ed9:2:11476), :19:20)
    at __run___mainLoop (eval at exports.evalCode (blob:https://screeps.com/cdbe6797-ff3f-4ad1-a7dd-efcc70585ed9:2:11476), :1:22556)
    at eval (eval at exports.evalCode (blob:https://screeps.com/cdbe6797-ff3f-4ad1-a7dd-efcc70585ed9:2:11476), :2:4)
    at Object.exports.evalCode (blob:https://screeps.com/cdbe6797-ff3f-4ad1-a7dd-efcc70585ed9:2:11476)
```

Wild guess
-----
Is this maybe because `Spawning` doesn't exist in the Screeps Javascript API but it's actually called `StructureSpawn.Spawning`?

**Comment by shanemadden** [comment]
Wild guess was dead on, thanks! Should be fixed in #276, though it'll likely be a bit before it's fully merged and released, ping if you need any assistance getting the fix incorporated locally in the interim.

**Comment by smessmer** [comment]
Cool, thanks for the quick fix. I'll try getting it by pointing my Cargo.toml dependency to GitHub master.
